.ci/run-tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# Function to display help
+show_help() {
+    echo "Usage: $0 [options]"
+    echo "Options:"
+    echo "  --force-recreate    Recreate and build containers before running tests"
+    echo "  --help              Show this help message"
+}
+
+# Check arguments
+force_recreate=false
+
+for arg in "$@"; do
+    case $arg in
+        --force-recreate)
+            force_recreate=true
+            ;;
+        --help)
+            show_help
+            exit 0
+            ;;
+        *)
+            echo "Unknown option: $arg"
+            show_help
+            exit 1
+            ;;
+    esac
+done
+
+# Run podman-compose only if --force-recreate is specified
+if [ "$force_recreate" = true ]; then
```

Comment:
Nit, can also do:

```
if [ -n "$force_recreate" ]; then
```

Reply:
I will keep the current logic.

`-n` only checks if the variable is non-empty, which is less specific than checking for the exact value true.

Even when in this script `force_recreate` is always defined at that point, `force_recreate` could hold in a future values like `false`, `maybe`, or `invalid`, but the condition `if [ -n "$force_recreate" ]; then` would evaluate as true for all of these, leading to incorrect behavior.